UPGRADING.md

```diff
@@ -3,6 +3,10 @@ Upgrading Hashie
 
 ### Upgrading to 4.0.0
 
+#### `Mash.load` option name changes
+
+The `whitelist_symbols` and `whitelist_classes` options on `Mash.load` are now named `permitted_symbols` and `permitted_classes`, to follow Ruby 2.6's lead on naming of those options. This does not change Hashie's compatibility with earlier versions of Ruby.
```

Comment:
If this was never released we don't need this. It wasn't right?

Reply:
Yeah, I think everything was set up to go for 3.7.0 but it was never pushed up to rubygems

Reply:
That's why I never increment versions during dev ;)